workspaces/tests/patch_state.rs

```diff
@@ -132,7 +133,7 @@ async fn test_patch_full() -> anyhow::Result<()> {
         .patch(&bob_id)
         .account(
             AccountDetailsPatch::default()
-                .balance(near_units::parse_near!("100 N"))
+                .balance(NearToken::from_near(100).as_yoctonear())
```

Comment:
Can `.balance` receive `NearToken`?

```suggestion
                .balance(NearToken::from_near(100))
```